brands/amenity/vending_machine.json

```diff
@@ -168,5 +168,45 @@
       "name": "Ключ здоровья",
       "vending": "water"
     }
+  },
+  "amenity/vending_machine|コカ・コーラ": {
+    "countryCodes": ["jp"],
+    "match": ["shop/vending_machine|コカコーラ"],
+    "nocount": true,
+    "tags": {
+      "amenity": "vending_machine",
+      "brand": "コカ・コーラ",
+      "brand:en": "Coca-Cola",
+      "brand:ja": "コカ・コーラ",
+      "brand:wikidata": "Q2813",
+      "brand:wikipedia": "ja:コカ・コーラ",
+      "drink": "cola",
+      "name": "コカ・コーラ",
+      "name:en": "Coca-Cola",
+      "name:ja": "コカ・コーラ",
+      "operator": "Coca Cola Japan Co., Ltd.",
```

Comment:
I am unsure about this one. Are you sure that every single vending machine dispensing Coca-Cola in Japan is operated by Coca Cola Japan Co., Ltd.?

Reply:
I my opinion, yes. We could ask a third party to confirm. Right?

Reply:
`The Coca-Cola system in Japan is comprised of Coca-Cola (Japan) Co, Ltd., which supplies concentrates, plans and develops new products, and conducts its marketing and advertising activities; system bottling partners, **which manufacture and sell products**, and other affiliated companies.  Coca-Cola (Japan) Co, Ltd. was established in 1957 as the Japanese subsidiary of The Coca-Cola Company`
If you mean that the store or market sells the product, then I could give you a reason. However, it is the company "provides the services" to its partners. That's why I tell you that it is Coca-Cola Japan as a main operator.

Reply:
Other quote about vending machines from [Coca-Cola website](http://j.cocacola.co.jp/peak_shift/about/):

> The development and rapid development of the peak shift vending machine of Coca-Cola, and unique public relations activities were evaluated, and the 2013 Energy Conservation Grand Prize (Energy Conservation Case Category) Economy and Industry Minister's Prize (Power Conservation Award) was received!
That means that Coca-Cola manufactures, sells and improves its machines. Even the company is plaing that all its machines are of low energy consumption.

Reply:
I think we're already sure who the operator is. I hope I'm not wrong